docs/topics/mta-cli-args.adoc

```diff
@@ -0,0 +1,210 @@
+|====
+| --explodedApp
+| --sourceMode
+| Neither Argument
+
+| The directory is evaluated as a single application.
+| The directory is evaluated as a single application.
+| Each subdirectory is evaluated as an application.
+|====
+
+File::
++
+[cols="1,1,1",options="header"]
+|====
+| --explodedApp
+| --sourceMode
+| Neither Argument
+
+| Argument is ignored; the file is evaluated as a single application.
+| The file is evaluated as a compressed project.
+| The file is evaluated as a single application.
+|====
+
+[id="cli-output-argument_{context}"]
+== Specifying the output directory
+
+Specify the path to the directory to output the report information generated by {ProductShortName}.
+
+.Usage
+[source,options="nowrap",subs="attributes+"]
+----
+--output <OUTPUT_REPORT_DIRECTORY>
+----
+
+* If omitted, the report will be generated in an `<INPUT_ARCHIVE_OR_DIRECTORY>.report` directory.
+* If the output directory exists, you will be prompted with the following (with a default of N).
++
+[source,options="nowrap",subs="attributes+"]
+----
+Overwrite all contents of "/home/username/<OUTPUT_REPORT_DIRECTORY>" (anything already in the directory will be deleted)? [y,N]
+----
+
+However, if you specify the `--overwrite` argument, {ProductShortName} will proceed to delete and recreate the directory. See the description of this argument for more information.
+
+[id="cli-source-argument_{context}"]
+== Setting the source technology
+
+A space-delimited list of one or more source technologies, servers, platforms, or frameworks to migrate from. This argument, in conjunction with the `--target` argument, helps to determine which rulesets are used. Use the `--listSourceTechnologies` argument to list all available sources.
+
+.Usage
+[source,options="nowrap",subs="attributes+"]
+----
+--source <SOURCE_1> <SOURCE_2>
+----
+
+The `--source` argument now provides version support, which follows the link:http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html[Maven version range syntax]. This instructs {ProductShortName} to only run the rulesets matching the specified versions. For example, `--source eap:5`.
+
+[WARNING]
+====
+When migrating to JBoss EAP, be sure to specify the version, for example, `eap:6`. Specifying only `eap` will run rulesets for all versions of JBoss EAP, including those not relevant to your migration path.
+
+See link:{ProductDocIntroToMTAGuideURL}/index#migration_paths_getting-started-guide[Supported migration paths] in _{IntroToMTABookName}_ for the appropriate JBoss EAP version.
+====
+
+[id="cli-target-argument_{context}"]
+== Setting the target technology
+
+A space-delimited list of one or more target technologies, servers, platforms, or frameworks to migrate to. This argument, in conjunction with the `--source` argument, helps to determine which rulesets are used. If you do not specify this option, you are prompted to select a target. Use the `--listTargetTechnologies` argument to list all available targets.
```

Comment:
`--listTargetTechnologies` is now `list-targets`

Reply:
Bulk update to fix